### DISCUSSION TITLE
Using http://opcfoundation.org/UA/ namespace higher than current 1.04 in the server

### DISCUSSION THREAD

**Discussion by plamenk** [FreeOpcUa/python-opcua#1525] [Q&A]
Dear community,
is there a way of overwriting the already included base OPC UA namespace on the server? Background is that some newer specs / namespaces are using the newer 1.05 version and their loading is failing.
Many thanks in advance.
Plamen

**Comment by schroeder-** [comment]
The library is deprecated. 
Use [opcua-asyncio](https://github.com/FreeOpcUa/opcua-asyncio) which also has a [sync-wrapper](https://github.com/FreeOpcUa/opcua-asyncio/blob/master/asyncua/sync.py), with very few changes in API.
